operator-wg/README.md

```diff
@@ -5,7 +5,7 @@ Create A Whitepaper (Initially Definition of an Operator)
 
 ### How to contribute
```

Comment:
I think we should add here a link to the `ONTRIBUTING.md`, WDYT?

Reply:
Added reference to CONTRIBUTING.md commit: https://github.com/cncf/sig-app-delivery/commit/82c0937b4a927cc7b6b6386e6e6ff8b56acee490